### PR TITLE
Add bencher for raw API

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

File: cmd/benchraw/main.go [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+	prometheus.MustRegister(txnRolledbackCounter)
+	prometheus.MustRegister(txnDurations)
+	http.Handle("/metrics", prometheus.Handler())
+
+	go http.ListenAndServe(":9191", nil)
+}
+
+// without conflict
+func batchRawPut(value []byte) {
+	cli, err := tikv.NewRawKVClient([]string{"localhost:2379"})
```
Comment (disksing) [inline cmd/benchraw/main.go]: Use `strings.Split(*pd, ",")` instead. The rest LGTM.

**Comment by disksing** [comment]
LGTM

**Comment by ngaut** [comment]
PTAL @siddontang

File: cmd/benchraw/main.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+	txnCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "tikv",
+			Subsystem: "txn",
+			Name:      "total",
+			Help:      "Counter of txns.",
+		}, []string{"type"})
+
+	txnRolledbackCounter = prometheus.NewCounterVec(
```
Comment (siddontang) [inline cmd/benchraw/main.go]: this metric is not used

File: cmd/benchraw/main.go [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	dataCnt   = flag.Int("N", 1000000, "data num")
+	workerCnt = flag.Int("C", 100, "concurrent num")
+	pdAddr    = flag.String("pd", "localhost:2379", "pd address:localhost:2379")
+	valueSize = flag.Int("V", 5, "value size in byte")
+
+	txnCounter = prometheus.NewCounterVec(
```
Comment (siddontang) [inline cmd/benchraw/main.go]: seem you don't need use MetricVec, using Counter is enough.

**Comment by siddontang** [comment]
Rest LGTM

**Comment by ngaut** [comment]
Removed all of the unnecessary metrics. It's much more clean now.
PTAL

**Comment by siddontang** [comment]
LGTM

**Pull request merged**